src/index.js

```diff
@@ -115,7 +138,11 @@ const getLikedPosts = (timestamp) => {
           for (let i = 0; i < likedPosts.length; i++) {
             const currentPost = likedPosts[i];
             if (Array.isArray(currentPost.photos)) {
-              currentPost.photos.forEach(pushToArray);
+              const filebase = cleanFileName(`${currentPost.blog_name}_${currentPost.timestamp}`);
```

Comment:
Sounds good and easy enough. I'll just adjust my code a bit on here and then for the cli tool.